codes/Action.ttl

```diff
@@ -38,6 +38,12 @@ idsc:AGGREGATE_BY_PROVIDER
     skos:note "This action is always evaluated at the provider side."@en ;
 .
 
+idsc:ADD
+    a ids:Action ;
+    rdfs:comment "This action modifies a number by adding a given value to it."@en ;
+    rdfs:label "add"@en ;
+    skos:note "This action modifies a number by adding a given value to it. The field to be modified and the given value are specified in the policy as idsc:SUBSET_SPECIFICATION and idsc:OPERAND, respectively."@en ;
```

Comment:
@changqin26 Could you please change the idsc:SUBSET_SPECIFICATION to idsc:JSONPATH/idsc:XPATH in the notes of the modify actions too?

Reply:
Thanks for pointing that out, already changed.

Reply:
The same change shall be applied to the definitions of other modify actions (divide, multiply, etc.)